core/pom.xml

```diff
@@ -468,6 +468,11 @@ THE SOFTWARE.
       <artifactId>commons-logging</artifactId>
     </dependency>
     <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
It was not explicitily defined, but rather pulled in as transitive dependency of acegi-security ([see JENKINS-4433] (https://issues.jenkins-ci.org/browse/JENKINS-4433?focusedCommentId=229977&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-229977)).